index/postings_test.go

```diff
@@ -197,35 +283,106 @@ func TestMultiMerge(t *testing.T) {
 
 func TestMergedPostings(t *testing.T) {
 	var cases = []struct {
-		a, b []uint64
-		res  []uint64
+		in []Postings
+
+		res Postings
 	}{
 		{
-			a:   []uint64{1, 2, 3, 4, 5},
-			b:   []uint64{6, 7, 8, 9, 10},
-			res: []uint64{1, 2, 3, 4, 5, 6, 7, 8, 9, 10},
+			in:  []Postings{},
+			res: EmptyPostings(),
 		},
 		{
-			a:   []uint64{1, 2, 3, 4, 5},
-			b:   []uint64{4, 5, 6, 7, 8},
-			res: []uint64{1, 2, 3, 4, 5, 6, 7, 8},
+			in: []Postings{
+				toPostings(),
+				toPostings(),
+			},
+			res: EmptyPostings(),
 		},
 		{
-			a:   []uint64{1, 2, 3, 4, 9, 10},
-			b:   []uint64{1, 4, 5, 6, 7, 8, 10, 11},
-			res: []uint64{1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11},
+			in: []Postings{
+				toPostings(),
+			},
+			res: toPostings(),
+		},
+		{
+			in: []Postings{
+				EmptyPostings(),
+				EmptyPostings(),
+				EmptyPostings(),
+				EmptyPostings(),
+			},
+			res: EmptyPostings(),
+		},
+		{
+			in: []Postings{
+				toPostings(1, 2, 3, 4, 5),
```

Comment:
what is blocking to use `newListPostings` everywhere?

Reply:
boilerplate

Reply:
how about if you change 
`newListPostings([]uint64) -> newListPostings(list ...uint64)`
than you can call it as `newListPostings(1, 2, 3, 4, 5)`
just tried it and it removes that boilerplate without the need for an extra func.


not a blocker though can leave it as if you prefer.

Reply:
It will increase this PR size, but did it, I like this idea :+1: